elements/pfe-modal/src/pfe-modal.scss

```diff
@@ -87,16 +107,20 @@ $LOCAL-VARIABLES: (
 	}
 
 	&__content {
-		overflow-y: auto;
+		overflow-y: pfe-local(Overflow, $region: overlay);
 		overscroll-behavior: contain;
-		max-height: pfe-local(MaxHeight);
+		height: pfe-local(MaxHeight);
 		padding: pfe-local(Padding);
 		box-sizing: border-box;
 
 		@media screen and (max-height: $pfe-modal--breakpoint--medium) {
-			@include pfe-print-local((
-				Padding: pfe-var(container-padding) calc(#{pfe-var(container-padding)} * 3) pfe-var(container-padding) pfe-var(container-padding)
-			));
+			--pfe-modal--Padding:  
```

Comment:
To prevent overwriting the padding variable, let's use your approach from the pfe-tabs PR:

```
@media screen and (max-height: $pfe-modal--breakpoint--medium) {
    padding: pfe-local(Padding, $fallback: pfe-var(container-padding) calc(#{pfe-var(container-padding)} * 3) pfe-var(container-padding) pfe-var(container-padding));
}
```

Reply:
updated